backend/templates/users/login.html

```diff
@@ -3,8 +3,75 @@
 {% load i18n %}
 {% load socialaccount %}
 {% block content %}
-<div class="container vh-100 d-flex flex-column justify-content-center">
-  <div class="mx-auto container-sm bg-body-secondary p-4 rounded-3">
+<style>
+  body {
+    background: url('../../media/backgrounds/login.jpg') repeat center center fixed;
+    background-size: contain;
+    color: white;
+  }
+
```

Comment:
Normalmente os styles vão dentro da tag head, acho que vale criar um block separado para elas e por la no base.html, algo como:

```suggestion
{% block styles %}
<style>
  body {
    background: url('../../media/backgrounds/login.jpg') repeat center center fixed;
    background-size: contain;
    color: white;
  }
...
{% endblock %}

{% block content %}
...
{% endblock %}
```